src/api/lib/tasks/auto_annotate_models.rake

```diff
@@ -0,0 +1,34 @@
+# NOTE: only doing this in development as some production environments (Heroku)
+# NOTE: are sensitive to local FS writes, and besides -- it's just not proper
+# NOTE: to have a dev-mode tool do its thing in production.
+if Rails.env.development?
+  task :set_annotation_options do
+    # You can override any of these by setting an environment variable of the
+    # same name.
+    Annotate.set_defaults({
+      'position_in_routes'   => "before",
+      'position_in_class'    => "before",
```

Comment:
@hennevogel, having it at beginning of the file could move all the class several lines down. But if this information is at the end, we will have all in its place and it will be easier to move into it. By the way, it is only a point of view, I can live with it 😄